src/pages/datenschutz.tsx

```diff
@@ -13,6 +13,12 @@ const DatenschutzPage = () => {
         perspiciatis amet provident a quaerat nemo suscipit quasi debitis
         impedit, fugiat officiis.
       </p>
+      <h2>Tracking Opt-Out (Matomo)</h2>
+      <iframe
+        title="Matomo Opt Out Tracking"
+        style={{ border: '0', height: '200px', width: '600px' }}
```

Comment:
@ohrie wir sind hier ja in der Tailwind-Welt. Das wäre also idealer Weise etwas wie `className="border-0 h-52 w-full` (oder `w-[600px]` wenn es genau das sein muss, was aber Mobile Probleme machen dürfte)

Reply:
Du hast recht! Ganz vergessen, hatte einfach den Code von Matomo übernommen. Der iframe macht so oder so Probleme auf mobile :/ Aber ich kenne keine andere Lösung.

Reply:
>Du hast recht! Ganz vergessen, hatte einfach den Code von Matomo übernommen. Der iframe macht so oder so Probleme auf mobile :/ Aber ich kenne keine andere Lösung.

https://developer.matomo.org/guides/tracking-javascript-guide#optional-creating-a-custom-opt-out-form sieht so aus, als ob wir das auch einfach bei uns selbst bauen könnten, oder?

Reply:
Gute Idee. Ich mache einen Issue daraus.